test/browser/plain.js

```diff
@@ -33,6 +33,21 @@ describe('browser with html with quotes in attributes', function() {
   });
 })
 
+describe("highlighBlock and data-language attribute", function() {
+  before(async function() {
+    await buildFakeDOM.bind(this)();
+  })
+  it('should set if unset', async function() {
+    this.hljs.highlightBlock(this.block);
+    this.block.dataset.language.should.equal("javascript")
+  });
+  it("should not alter if already set", async function() {
+    this.block.dataset.language="existingData";
```

Comment:
```suggestion
    this.block.dataset.language = "existingData";
```

Reply:
Lol

Reply:
We should consider applying [prettier](http://prettier.io) via ESLint to our tests and code to maintain a consistent coding style.

Reply:
Perhaps. You could open an issue. 😉. I’ve been using JSLint some but it doesn’t yell at that level I don’t think. I try but don’t get hung up on tiny things like that too much. 

How would you make that automatic? Like add it as a CI check?

Reply:
Yeah, CI check. We would just add on TravisCI `eslint .` and few other dev dependencies to package.json ... a few style rules.

Reply:
I’ll play around with it locally first.

Reply:
Ok, if you need want to see how it's used in a different project:
https://github.com/w3c/respec/

See the .travis file and package.json... you can also try `eslint --fix` to see how many changes there would be.